src/hammer-vlsi/hammer_vlsi/constraints.py

```diff
@@ -237,6 +230,7 @@ def to_dict(self) -> dict:
     ('no_connect', Optional[bool]),
     ('x', Decimal),
     ('y', Decimal),
+    ('group', Optional[str]),
```

Comment:
What is this about?

Reply:
This is for grouping pins into schematic symbols